earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -127,6 +127,8 @@ AssetVersionImplD::StateChangeNotifier::SendNotifications(
       notify(NFY_WARN, "'%s' has broken %s '%s'",
              assetVersion->GetRef().c_str(), typeName.c_str(), ref.c_str());
     }
+
+    i++;
```

Comment:
Preference: Although, it would not make difference in this case, my style preference is to rewrite the range `for` to a regular `for` with an explicit iteration expression, and add `i++` to that expression.  That way, you don't have to worry about `continue` statements making the counter out of sync.